### ISSUE TITLE
Restrict email addresses?

### ISSUE THREAD

**Issue by anandcsingh** [anandcsingh/3for3#20]
Can a person only submit once?

Should we verify their email after pledging?

**Comment by pedextt** [comment]
I dont think we should necessarily restrict to one submit, maybe the person decided to take on more pledges? Perhaps we should check if the pledger has pledged before and associate the new pledges with the same id, with a meaningful message reminding them that they've made the pledge before/new pledges saved.

-----Original Message-----
